content/material_and_questions.md

```diff
@@ -44,4 +53,151 @@ In the below table, any minimum or maximum variation between material years are
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
+The Question Type abbreviation is "CR"
+
+##### Multiple-Answer, Chapter (only)
+Must have more than one answer.
+The Question Type abbreviation is "MACR"
+
+#### Quote
+Will be direct quotes from Scripture
+Must be quoted word-perfect.
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The entire reference is part of the required question + answer.
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
+
+##### Quote
+A single verse
+The Question Type abbreviation is "QT"
+
+##### Quote-these-Two-Verses
+Two consecutive verses
+The Question Type abbreviation is "Q2V"
+Should be verses that make more sense when quoted together.  Example: Gal 5:22-23 
+
+#### Finish
+Will be direct quotes from Scripture
+Must be quoted word-perfect
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
```

Comment:
I used the language from QTs ("must be spiritually significant") to be consistent, and I did not use the existing language under Finishes ("Must be of spiritual value or be significant to the chapter.")  Thoughts?

Reply:
Here's another area we should change to be less inconsistent. In this case, I think we should go with the full "must be of spiritual value or be significant to the chapter" for both sections instead, since otherwise we're technically removing stuff from the rulebook. But one way or another, the two sections need to be consistent.

Reply:
Yeah, this was a spot where I was editorializing, because I HATE "significant to the chapter."  That's everything. Which means ANY wording on significance is null and void and shouldn't exist. (This is me speaking on the current wording of the rule book--I absolutely believe F/Q should have some "significance threshold implying that it applies to < 100% of the verses")

Reply:
I recommend:

> must be of spiritual value **and** be significant to the chapter

(The bold-ing added here only to show a difference from what Jeremy wrote, not to suggest I think it should be in the final copy.)

Reply:
changing OR to AND is not functionally equivalent.

Reply:
I'd argue it is func equiv to what IBQ does. The different language between types is what I'd call a bug in the current rulebook.

Reply:
ummm, IBQ ignores these altogether.

Reply:
Oy. True. Ugh. OK, to resolve, I think I'm going to have to suggest an autocratic override:

**Proposal:** Bugs in the rulebook can be fixed and we're still functionality equivalent. And bugs at IBQ can also be fixed and we're still functionality equivalent. But the bugs have to be obvious bugs, not just misinterpretations or style differences. So stuff that IBQ completely ignores is an IBQ bug.

Reply:
I like your autocratic decree.

And I especially like the outcome, because IBQ needs to deal with the inconsistency. I've been saying it for almost a decade now, either remove the rulebook language on { stand on its own, spiritually significant, significant to the chapter }, or follow what the rule book says { through robust IBQ question writing and editing processes }.

Keeping the rule book language and ignoring it (to varying degrees) is just a giant middle finger to quizzers.